### PR TITLE
Introduces automatic refresh setting for balances and prices

### LINKED ISSUE #916: Refresh some master data from the backend periodically
## Motivation

Because users do not always close/re-open their apps, certain backend data should be refreshed periodically. 

## Specification

Let's make a list of data that is grabbed from the backend that needs to be refreshed periodically:

- [x] Currency Exchange Rates (every 30 mins / 1 hr?)
- [x] Blockchain Balances
- [x] Exchange balances 
- [ ] Trades

### PR DESCRIPTION
**Pull request by kelsos**
Closes #916 

- Autorefreshes blockchain/looopring/manual/exchange balances and prices
- Selectable options are none/30m/1h/2h
- Remaining to add auto-refresh of trades if the user is looking at the trades

### CODE REVIEW

File: frontend/app/src/locales/en.json [hunk 1]
```diff
@@ -614,6 +614,46 @@
     "default_timeframe": "Dashboard graph default timeframe",
     "default_timeframe_description": "Set the default time frame for the dashboard graph. This timeframe will be pre-selected upon login. By default it will remember the previous session's selection."
   },
+  "frontend_settings": {
+    "hint": {
+      "refresh": "Select how often your balances will be automatically refreshed"
+    },
+    "label": {
+      "query_period": "Query period in seconds",
```
Comment (LefterisJP) [inline frontend/app/src/locales/en.json:622]: Perhaps not seconds but minutes as discussed?

**Review by LefterisJP — COMMENTED** [review]
I can't really test this myself as it makes no difference which account tests it. Just make the changes we discussed and perhaps post a screenshot of the setting here.

File: frontend/app/src/locales/en.json (hunk 1 above)
Comment (kelsos) [reply frontend/app/src/locales/en.json:622]: Query Period = Periodic Query Period

**Comment by kelsos** [comment]
image

**Review by LefterisJP — APPROVED** [review]
LGTM sir.

**Pull request merged**